document/chapters/02-background.tex

```diff
@@ -929,14 +929,109 @@ \subsection{Signed Distance Fields}
+
+The bulk of Monte Carlo memory consumption in native calculations is attributed
+to nuclear cross-section data and tally data structures with the analytic CSG
+geometry representation taking up a small portion of the overall program memory.
+
+\subsection{Cross-Section Data}
+
+Cross-section data is necessary to represent the physical
+processes governing particle transport through the virtual geometry. Various
+formats of this data exist with both continuous and discrete representation of
+cross section information. Continuous representations of data are compact in
+terms of memory requirements, but are more computationally expensive to evaluate
+when sampling these probability distribution functions. Discrete representations
+of this data consume more memory in simulation, but require less computation
+when extracting data. The amount of memory consumed for nuclear data is
+dependent on the number of unique materials used in the problem and material
+composition as well. This data can occupy the majority of simulation memory for
+large problems with many materials.
+
+\subsection{Tally Data Structures}
+
+Results of the Monte Carlo calculation are kept as tallies of particle
+contributions to physical quantities at various locations in the physical
+model. Tallies on surfaces or in cells change the memory usage of the simulation
+insignificantly, unless additional data is needed to calculate derived
+quantities. Mesh-based tallies on the other hand can dominate the memory
+usage. In MCNP and many other Monte Carlo codes, mesh tallies are defined by the
+user as structured grids in either Cartesian, cylindrical, or spherical
+coordinates. These tallies are often used for better spatial resolution of
+physical data in the simulation and for coupling to analysis in other
+engineering domains. Tetrahedral mesh tallies are also supported by several
+Monte Carlo codes DAGMC interacts with, including MCNP
+\cite{LANL_MCNP5_VOLIII}. The size of these tallies vary based on the needs of
+the user and the trade-offs associated with spatial resolution of solutions and
+statistical convergence related to mesh element sizes.
+
+\subsection{Parallel Computation Schemes}
+
+For the Monte Carlo code used in this work, MCNP, the parallelism method applied
+is a master/slave scheme standard for many applications using the Message
+Passing Interface (MPI) \cite{Forum_1994}. The master process is responsible for
+initializing the problem, determining the load balance for the number of
+parallel processes requested by the user, and collecting/accumulating
+information at the end of the simulation. In MCNP, particle histories are
+divided into equal segments among the slave processes for evaluation. More
+information on the method for particle history division and random number
+sequence assignment was detailed by Deng and Xie \cite{Deng_1999}. For each
+slave process, a corresponding set of the nuclear data, tally data structures,
```

Comment:
Not sure you need to say so much about this before this sentence... maybe combine with the next section?

Reply:
Did these changes look okay?

Reply:
:+1: